pkg/devcontainer/crane/run.go

```diff
@@ -0,0 +1,96 @@
+package crane
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/loft-sh/log"
+)
+
+var (
+	craneSigningKey string
+)
+
+const (
+	PullCommand    = "pull"
+	DecryptCommand = "decrypt"
+
+	GitCrane = "git"
+
+	BinPath = "devpod-crane" // FIXME
+
+	tmpDirTemplate = "devpod-crane-*"
+)
+
+type Content struct {
+	Files map[string]string `json:"files"`
+}
+
+func IsAvailable() bool {
+	_, err := exec.LookPath(BinPath)
+	return err == nil
+}
+
+func runCommand(command string, args ...string) (string, error) {
+	cmd := exec.Command(BinPath, append([]string{command}, args...)...)
```

Comment:
why not simply `cmd := exec.Command(BinPath, command, args...)`?

Reply:
oh yeah now I remember: 
```
pkg/devcontainer/crane/run.go:40:40: too many arguments in call to exec.Command
        have (string, string, []string)
        want (string, ...string)
```